### DISCUSSION TITLE
[Feature] link to breaking changes list in update notification

### DISCUSSION THREAD

**Discussion by MaxMatti** [immich-app/immich#10876] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Whenever there's a new release available I get a notification when visiting the website. My workflow then consists of closing it, going to the installation tutorial for docker-compose, clicking [this link](https://github.com/immich-app/immich/discussions?discussions_q=label%3Abreaking-change+sort%3Adate_created) in the red box near the bottom to check whether it's a breaking change and then running the command to rebuild/update my docker-compose setup.

As I have to check that link every time anyway, would it be possible to just include it (or the information whether any change is breaking) in the update notification?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
If you click the link in the notifications that takes you to the release notes, those say pretty clearly at the top when it's a breaking change.

**Reply by MaxMatti** [reply]
Unfortunately it only takes me to the latest release which does not show all changes. For example releases v1.109.0, v1.109.1 and v1.1.09.2 all came within a day, so I only saw the release notes for v1.109.2 even though I was still on v1.108.0. Thankfully there were no breaking changes in v1.109.0 this time, but this is not always the case, for example v1.105.0 and v1.105.1 were released on the same day with the former introducing breaking changes and the latter not mentioning them and even containing a note saying that you should update immediately.
